packages/nodejs/src-getaddrinfo.patch

```diff
@@ -0,0 +1,48 @@
+--- /dev/null	1972-07-31 19:09:33.659999999 +0530
```

Comment:
This is fix for getaddrinfo on some devices where querying for localhost with IPv6 fails with EA_NODATA. A bug in netd somewhere